src/renderer/widgets/IWidget.cpp

```diff
@@ -82,7 +83,8 @@ static std::string getTime() {
 
 IWidget::SFormatResult IWidget::formatString(std::string in) {
 
-    char* username = getlogin();
+    auto uidPassword = getpwuid(getuid());
+    char* username = uidPassword->pw_name;;
```

Comment:
double ; here on accident I believe

Reply:
Thanks for your attention to detail! It's fixed now